File: taller/AMarotoChivite-taller/AMarotoChivite-taller-Java/src/test/java/simulacionTaller/factories/WorkerFactoryTest.java
package simulacionTaller.factories;

import org.junit.jupiter.api.Test;
import simulacionTaller.models.Worker;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class WorkerFactoryTest {

    @Test
    void createTest() {
        Worker worker = WorkerFactory.create();
        // No nulo, para ver si se han asignado correctamente los datos
        assertNotNull(worker);
    }
}

File: taller/AMarotoChivite-taller/AMarotoChivite-taller-kotlin/src/test/kotlin/simulacionTaller/factories/WorkerFactoryTest.kt
package simulacionTaller.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class WorkerFactoryTest {
    @Test
    fun createTest() {
        val worker = WorkerFactory.create()
        // No nulo, para ver si se han asignado correctamente los datos
        Assertions.assertNotNull(worker)
    }
}